modules/core/src/main/java/org/apache/ignite/internal/tostring/IgniteToStringBuilder.java

```diff
@@ -2214,6 +2215,28 @@ private static void incValues(IdentityHashMap<Object, EntryReference> svdObjs, O
         }
     }
 
+    /**
+     * Converts Tuple to string representation.
+     *
+     * @param tuple Tuple.
+     * @return String.
+     */
+    public static String tupleToString(Tuple tuple) {
+        // Keep the same as TupleImpl.toString().
```

Comment:
If this is important, and we can not move this logic to a single function by some reason, let's at least add the test that compares string representation of these implementations.

Reply:
`ignite-api` can't depend on `ignite-core` and we decided not to have internal classes in `ignite-api`, so this has to be duplicated.

Reply:
Added `testToStringMatchesTupleImpl`.